src/ansys/dynamicreporting/core/utils/report_objects.py

```diff
@@ -1465,7 +1465,6 @@ def get_json_key_limits(cls):
         d = super().get_json_key_limits()
         d["name"] = 255
         d["report_type"] = 50
-        d["params"] = 4096
```

Comment:
yeah, hardcoded limits like these should be done away with and ideally the server should tell you if you have exceeded the limit in any case @margalva